v1/client_test.go

```diff
@@ -30,18 +37,32 @@ func ExampleNewClient() {
 	log.Println(client)
 }
 
-func TestWriteSchemaCall(t *testing.T) {
-	t.Parallel()
-	require := require.New(t)
+func randomString(length int) (string, error) {
+	buffer := make([]byte, length)
+	_, err := rand.Read(buffer)
+	if err != nil {
+		return "", err
+	}
+	return base64.StdEncoding.EncodeToString(buffer)[:length], nil
+}
 
-	// TODO: should we get a handle on the connection in order to be able to close it?
-	// It should only matter in testing, but it could still be a problem.
+func testClient(t *testing.T) *authzed.Client {
```

Comment:
if you add a call to `t.Helper()` here, failures in this method will report as failures in the callsite in the calling test, which can help when code like this is shared between a bunch of tests